src/Test.FunctionalTests.Sarif/Multitool/BaselineOptionTests.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Mime;
+using System.Text;
 
 using FluentAssertions;
```

Comment:
Esp. since there are some new files here and you're a fellow appreciator: have you measured code coverage?  If so: where did it land?